easybuild/easyconfigs/p/PCMSolver/PCMSolver-1.2.3-foss-2018b-Python-3.6.6.eb

```diff
@@ -0,0 +1,42 @@
+easyblock = 'CMakeMake'
+
+name = 'PCMSolver'
+version = '1.2.3'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://pcmsolver.readthedocs.org'
+description = """An API for the Polarizable Continuum Model."""
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+#toolchainopts = {'cstd': 'c99'}
+
+source_urls = ['https://github.com/PCMSolver/pcmsolver/archive/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['d1ef6bdc268a2e719b36c85125c3543df7a26e1a738daf4483f4ded0c76f5b60']
+
+dependencies = [
+    ('Python', '3.6.6'),
+    ('zlib', '1.2.11'),
+    ('Boost', '1.67.0'),
+]
+
+builddependencies = [
+    ('CMake', '3.12.1'),
+    ('Eigen', '3.3.7', '', True),
+]
+
+configopts = '-DCMAKE_BUILD_TYPE=Release -DEIGEN3_ROOT=$EBROOTEIGEN'
+configopts += ' -DCMAKE_CXX_FLAGS="$CXXFLAGS $LIBLAPACK_MT"'
```

Comment:
@tovrstra This is the culprit for the problem you're seeing...

I'm not sure why we have this in the first place, but it's no longer needed, build is fixed by dropping this, see https://github.com/tovrstra/easybuild-easyconfigs/pull/5

Reply:
I mindlessly copied it from an older recipe. Sorry...